### ISSUE TITLE
Auto Loading samples in QuadSimpler

### ISSUE THREAD

**Issue by molluskderek** [nysthi/nysthi#50]
.65 no longer crashes on attempting to load samples, but samples no longer auto load when opening a saved project. It does, at least, give the filename so we can manually load again but it would save tons of time if it could auto load again. Really sweet module!

**Comment by nysthi** [comment]
are you sure ?
I use OSX 10.13.5 and is working fine

**Comment by molluskderek** [comment]
Yeah, definitely. My samples are on a network drive though, so I'll double check with a local file. But it was working before and then stopped after .65

**Comment by nysthi** [comment]
samples are never stored remotely !
any time you import, a special copy is created in the /RAck/plugins/nysthi/cache
directory
do you have the directory ?

**Comment by molluskderek** [comment]
That's what happened, then. When I updated to .65 I replaced the whole nysthi folder in plugins, so I'll recover that cache folder from backup.

Is there any way to have them auto-load if they're not in the cache? I see it being a problem long-term because over time I will have imported maybe hundreds of samples, all of which are now sitting on the cache folder of an SSD drive. I see the purpose of having the cache folder, absolutely, but maybe it should have a limit? Maybe it does already, I don't know.

**Comment by antoniotuzzi** [comment]
would be solved if I move the cache directory into the "Rack" directory
but people would be not happy to have their directory polluted by me.
Or maybe Rack should offer a common "cache" directory, where developer could create personal folder that could be shared between different installations 
I'm going to talk again to Andrew

**Comment by nysthi** [comment]
about the limits:
if you delete a Simpler, the related files are deleted too

but if you close a project that contain a simpler
and after you trash the project
the cache folder will contain no more used sampled

can't control it !

**Comment by molluskderek** [comment]
In my case, all my samples for Kontakt, etc. are done with VEPro over the network, so that's where my samples are. Either way, I guess NYSTHI doesn't know that it's a network folder. Having an overall Rack cache folder is a great idea, as long as we have some control over how much space it takes up. I know a lot of people are using laptops and I'm on a mac pro with SSD. And again, I guess it would be great if Simpler could still link to the original samples somehow in case they are missing from cache. Thanks for the great modules!

**Comment by nysthi** [comment]
could be an idea (feature request), I have to check
(maintain an original reference in case of missing cache)

**Comment by patman023** [comment]
I could swear I saw it (but google-fu is failing me) - is this not a v1.0 milestone?

**Comment by nysthi** [comment]
> I could swear I saw it (but google-fu is failing me) - is this not a v1.0 milestone?

something similar, but is it is something I have to consider for my sampler
(would be possible to consolidate some of the sample with the project file)

**Comment by patman023** [comment]
https://github.com/VCVRack/Rack/issues/1175
